docs/contributing.rst

```diff
@@ -461,6 +460,7 @@ Here's a generic docstring template::
          Write a short description of parameter my_param_1.
       my_param_2 : str, {'vector', 'matrix'}
          Write a short description of parameter my_param_2.
+         Optional, default: 'vector'.
```

Comment:
what about the convention that when listing the possible values of an optional parameter, the default is the first one to appear, that is, {'vector', 'matrix'}, optional implies default: 'vector'?